lib/voicebase/v2/client.rb

```diff
@@ -26,25 +26,44 @@ def authenticate!
       end
 
       def upload_media(args = {}, headers = {})
+
+        media_url = require_media_file_or_url(args)
+
+        # test URL from VoiceBase
+        # media_url = 'https://s3.amazonaws.com/voicebase-developer-test-content-dev/mpthreetest.mp3'
+
         form_args = {
-            'media': require_media_file_or_url(args),
-            'configuration': {
-                'executor': 'v2'
+            'media' => media_url,
+            'configuration' => {
+                'configuration' => {
+                    'executor' => 'v2'
+                }
             }
         }
+
+        # external ID is only partially supported in the V2 API (can't get plain text transcripts or delete media)
         if args[:external_id]
           form_args.merge!({
-            'metadata': {metadata: {external: {id: "'#{args[:external_id]}'"}}}
-          })
+                               'metadata' => {
+                                   'metadata' => {
+                                       'external' => {
+                                           'id' => "#{args[:external_id]}"
+                                       }
+                                   }
+                               }
+                           })
```

Comment:
I'm not sure which ones can be converted to symbols and which need to remain as strings. The original code as I received it used both. I know it works now as-is. Do you see a strong reason to change it @kblake?